internal/cobraprompt/prompt.go

```diff
@@ -76,6 +76,13 @@ func handleExit() {
 	}
 }
 
+var SuggestionColorOptions = []prompt.Option{
+	prompt.OptionSelectedSuggestionTextColor(prompt.White), prompt.OptionSuggestionTextColor(prompt.White),
+	prompt.OptionSelectedDescriptionTextColor(prompt.LightGray), prompt.OptionDescriptionTextColor(prompt.LightGray),
+	prompt.OptionSelectedSuggestionBGColor(prompt.Blue), prompt.OptionSuggestionBGColor(prompt.DarkGray),
+	prompt.OptionSelectedDescriptionBGColor(prompt.Blue), prompt.OptionDescriptionBGColor(prompt.DarkGray),
+}
+
```

Comment:
I guess this is a change we want to do but can't do due to nonintegrated go-prompt? Why do we have this change? I mean whole pr is about integrating a library except these lines.

Reply:
Yes you are right, scope of the the PR changed along the way. How about I revert these line and address them in another PR? Do you think that would be more appropriate?

Reply:
We can have them in this PR too, but I wondered the purpose of these lines

Reply:
Thanks for pointing that 🙏. I also updated the PR description with this small explanation.

Purpose of that is:
Changing prompt styling to more intuitive colors since it creates trouble for users specially when there are only two suggestions prompted.